src/item.cpp

```diff
@@ -1583,8 +1615,10 @@ std::string item::info(bool showtext, std::vector<iteminfo> &dump_ref) const
                 }
             }
             if (known_recipes.size() > 24) {
+                dump->push_back(iteminfo("DESCRIPTION", "--"));
```

Comment:
I could add it above (in the damage preview) or have two sections next to each other.
I don't really see the reason for omitting the newline/"--" before this section. I'd say it looks better with it.